corehq/ex-submodules/casexml/apps/phone/utils.py

```diff
@@ -38,6 +38,9 @@ def delete_sync_logs(before_date, limit=1000, num_tries=10):
     raise CouldNotPruneSyncLogs()
 
 
+def delete_sql_synclogs(before_date):
+    SyncLogSQL.objects.filter(date__lt=before_date).delete()
```

Comment:
seems ok for now but we could also just drop one of the partition tables once per week

Reply:
For now we have to construct the [tablename manually](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/phonelog/tasks.py#L15-L24)

Would be great to PR to the upstream project a way to expose table names. I think the relevant code is [here](https://github.com/maxtepkeev/architect/blob/master/architect/databases/postgresql/partition.py#L151-L178)

Note that we will get degraded performance for having a ton of empty tables, but it likely won't be an issue for a year or so